k8s/perftest/common/ia/case-notifications-api.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: case-notifications-api
+  namespace: ia
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: ia-case-notifications-api
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ia-case-notifications-api
+    version: 0.0.2
+  values:
+    java:
+      replicas: 2
+      memoryRequests: '768Mi'
```

Comment:
if you update your docker base image you can remove the this and the next 3 lines